### PR TITLE
Make docs logo clickable

### PR DESCRIPTION
**Pull request by shsethi**
#### PR checklist

- [x] Addresses an existing issue: fixes [#93](https://github.com/palantir/blueprint/issues/93)

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @shsethi! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/docs/src/components/navbar.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class Navbar extends React.Component<INavbarProps, {}> {
         return (
             <div className="pt-navbar docs-navbar docs-flex-row">
                 <div className="pt-navbar-group">
-                    <div className="docs-logo" />
+                    <a href="./"><div className="docs-logo"/></a>
```
Comment (giladgray) [inline packages/docs/src/components/navbar.tsx]: go ahead and replace the `div` with the `a` so there's only one element

**Comment by llorca** [comment]
@shsethi thanks for the contrib! 💯  I opened another PR (https://github.com/palantir/blueprint/pull/177) for it cause we'd like to merge it now and push it live

**Comment by shsethi** [comment]
When you come really close to getting your first open source PR merged but not close enough 😅

**Pull request closed without merging**